src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1317,24 +1320,41 @@
        (str/join ":")
        (str/upper-case)))
 
+(defn get-certificate-details
+  "Return details from a X509 certificate."
+  [^sun.security.x509.X509CertImpl cert]
```

Comment:
Elsewhere in this file, instead of doing type hints we would do something like:
```clj
(schema/defn get-certificate-details
  "..."
  [cert :- X509CertImpl]
  (...))
```
Where `sun.security.x509.X509CertImpl` had already been imported in the `ns` and could we referenced just by the class name here.

Reply:
Fixed, thanks!